### PR TITLE
Adding the file test.spec.js

### PR DESCRIPTION
**Pull request by nagarajanqa**
Adding the New file test.spec.js from QA branch to  main branch

### CODE REVIEW

File: cypress/integration/test.spec.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/// <reference types="Cypress" />
```
Comment (KaviarasuRajamanickam) [inline cypress/integration/test.spec.js:1]: Hope this line represents the cypress default

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
```
Comment (KaviarasuRajamanickam) [inline .gitignore:1]: Changes look good

**Review by KaviarasuRajamanickam — CHANGES_REQUESTED** [review]
Please update the changes and revert back

File: cypress/integration/test.spec.js (hunk 1 above)
Comment (nagarajanqa) [reply cypress/integration/test.spec.js:1]: We have updated the code as per the review comments

**Review by KaviarasuRajamanickam — APPROVED** [review]
Approved

**Pull request merged**